<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyCalc</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="OldOperationsList">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="uniformItemSizes">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="TextResult">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="BaseOutSelector">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <item>
         <property name="text">
          <string>Dec</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Oct</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hex</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bin</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="MemSizeSelector">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Integer</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Float</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Double</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Byte (8 bits)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Word (16 bits)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Double Word (32 bits)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Quad Word (64 bits)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Oct Half Word (6 bits)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Oct Word (12 bits)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Oct Double Word (24 bits)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Oct Triple Word (36 bits)</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CheckSigned">
        <property name="text">
         <string>Signed</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="Line" name="line_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Endianness</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="EndianLittle">
          <property name="text">
           <string>Little</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">GroupEndianness</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="EndianBig">
          <property name="text">
           <string>Big</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">GroupEndianness</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="EndianNative">
          <property name="text">
           <string>Native</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">GroupEndianness</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="Line" name="line_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Binary Format</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QRadioButton" name="FormatMS">
          <property name="text">
           <string>MS</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">GroupBinFormat</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="FormatC1">
          <property name="text">
           <string>C1</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">GroupBinFormat</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="FormatC2">
          <property name="text">
           <string>C2</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">GroupBinFormat</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QToolButton" name="InputSettingsButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="BaseInSelector">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <item>
         <property name="text">
          <string>Dec</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Oct</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hex</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bin</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="TextInput"/>
      </item>
      <item>
       <widget class="QPushButton" name="OKButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>323</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="MenuAddFunction"/>
    <addaction name="MenuCalcString"/>
    <addaction name="MenuColorPicker"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="MenuAddFunction">
   <property name="text">
    <string>Add custom function...</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Offset calculator...</string>
   </property>
  </action>
  <action name="MenuCalcString">
   <property name="text">
    <string>String encoding...</string>
   </property>
  </action>
  <action name="SetClassic">
   <property name="text">
    <string>Light</string>
   </property>
  </action>
  <action name="SetDark">
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="MenuColorPicker">
   <property name="text">
    <string>Color picker...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="GroupEndianness"/>
  <buttongroup name="GroupBinFormat"/>
 </buttongroups>
</ui>
